### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by bboehmke** [containrrr/shoutrrr#183] [Q&A] [answered]
Hi,
it has been some time now since the last release of shoutrrr.
Is there any estimation when there will be the next release?

It would be great to use watchtower with the new matrix notification support.

**Comment by piksel** [comment, marked as answer]
Shoutrrr itself has already been released with matrix support, but Watchtower has not been updated to use it.
That shouldn't be too far off once the vacations are over, but it's hard to give time estimates for volunteer community projects. Sorry.

**Reply by bboehmke** [reply]
What I mean is a version that was tagged in git.
The last one is from April and the Matrix changes where added in May so they are not part of v0.4.4.

**Reply by piksel** [reply]
Yeah, you are correct. I will release v0.5 as soon as the outstanding bugs have been fixed: https://github.com/containrrr/shoutrrr/milestone/7

**Reply by bboehmke** [reply]
sounds perfect

**Reply by piksel** [reply]
v0.5 is now released! 🎉

**Reply by bboehmke** [reply]
cool, thanks

**Reply by simonaronsson** [reply]
Although, this still does not mean they will be included in watchtower until a new release has been cut, but as @piksel said, that shouldn't be too far off.
